Lib/test/test_imaplib.py

```diff
@@ -497,6 +497,56 @@ def test_with_statement_logout(self):
 
     # command tests
 
+    def test_idle_capability(self):
+        client, _ = self._setup(SimpleIMAPHandler)
+        with self.assertRaisesRegex(imaplib.IMAP4.error,
+                'does not support IDLE'):
```

Comment:
CI failures are a result of this failing test:
```suggestion
                'does not support IMAP4 IDLE'):
```

Reply:
Thank you. It has been so long since I wrote that test that it didn't jump to mind when you suggested the change, and the big merge at about the same time seemed more likely to have been the cause.